cmd/main.go

```diff
@@ -269,6 +274,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	// If configured to managed CRDs, do a preliminary install of them here. The Installation controller
+	// will reconcile them as well, but we need to make sure they are installed before we start the rest of the controllers.
+	if bootstrapCRDs || manageCRDs {
```

Comment:
If I understand it correctly both  bootstrapCRDs and manageCRDs will install the OSS CRDs on startup. When manageCRDs is specified, the core controller will also apply the CRDs based on the variant.
What I don't fully understand at this point is that if my memory serves me right, the controller runtime requires a full restart in order to use new CRDs. In the case of enterprise, how can we use these flags to allow us to stop applying manifests manually?

Reply:
> What I don't fully understand at this point is that if my memory serves me right, the controller runtime requires a full restart in order to use new CRDs.

I'll try this out.. I didn't notice this problem when testing my changes with OSS Calico.

Reply:
I sort of assume we would have handled that already with the existing `manageCRDs` logic though.

Reply:
I checked and didn't see a restart in the core controller.

Reply:
There is this logic later in the reconcile loop that may solve the issue I brought up: https://github.com/tigera/operator/blob/master/pkg/controller/installation/core_controller.go#L968